### ISSUE TITLE
Inconsisten version - Github Tags vs. _version.py 

### ISSUE THREAD

**Issue by jan-janssen** [materialsproject/emmet#564]
The `_version.py` files are always one version behind, for example: 
https://github.com/materialsproject/emmet/blob/v0.38.0/emmet-core/_version.py

**Comment by munrojm** [comment]
Yes, unfortunately this will always be the case. We only use the version files for easy internal version detection in our code deployment pipelines.

That being said, the code in `main` will always be up to date with the latest release. Additionally, this should not affect users as code like

```python
from emmet.core import __version__
```

will always show the correctly installed version.

**Comment by jan-janssen** [comment]
But it affects `pip check`.

**Comment by munrojm** [comment]
Can you provide me more details about what issue you are running into?

**Comment by munrojm** [comment]
Perfect, thank you. I will take a look ASAP.

**Comment by munrojm** [comment]
One more change on my end. Once things look good I will close this issue.

**Comment by munrojm** [comment]
Reverted for now to fix deployment issues. Re-opening this until a new solution is found.

**Comment by jan-janssen** [comment]
The issue I have with the latest version is that the version is generated based on the git tag: 
```
emmet-core 0.1.dev76+g7b6631a.d20221023.
```
rather than 
```
emmet-core 0.36.4
```

**Comment by munrojm** [comment]
What is that messing up for you specifically?

**Comment by jan-janssen** [comment]
https://github.com/conda-forge/emmet-core-feedstock/pull/18

**Comment by munrojm** [comment]
Hm, okay I will take a look.

**Comment by jan-janssen** [comment]
It seems this issue was fixed with the release of `0.38.9`.
